tests/examples/test_dataset_loader.py

```diff
@@ -69,10 +69,8 @@ def add_to_dict(func: str, dataset_function: Callable[[], Any]):
 
     # Collect all `download_<name> or `load_<name>` functions
     def _is_dataset_function(name, item):
-        return (
-            isinstance(item, FunctionType)
-            and name.startswith('download_')
-            or name.startswith('load_')
+        return (isinstance(item, FunctionType) and name.startswith('download_')) or name.startswith(
+            'load_'
         )
```

Comment:
```suggestion
        return isinstance(item, FunctionType) and (
            name.startswith('download_') or name.startswith('load_')
        )
```

Not sure if this is correctly formatted with tabs.

Reply:
name.startswith can accept a tuple input and ruff will complain about thist, see my suggestion above

Reply:
This was fixed in https://github.com/pyvista/pyvista/pull/6915/commits/8e574e338d91b91ffc4d237baabfba5c4337b098